lib/src/abstract_tracer.dart

```diff
@@ -54,6 +55,15 @@ abstract class AbstractTracer {
   ///     var serverSpan = Tracer.startSpan('...', { childOf : wireCtx });
   SpanContext extract(String format, dynamic carrier);
 
+  /// Returns the current [ScopeManager], which may be a noop but may not be
+  /// null.
+  ScopeManager get scopeManager;
+
+  /// Returns the activer [Span]. This is a shorthand for
+  /// `Tracer.scopeManager().active().span()` and null will be returned if
+  /// [ScopeManager].active()} is null.
+  Span activeSpan();
```

Comment:
Are these changes missing the [Tracer Changes](https://github.com/opentracing/specification/blob/master/rfc/scope_manager.md#tracer-changes) from the spec?

Also, is there supposed to be an `activeSpan` on this class? I don't see that in the spec

Reply:
> Are these changes missing the Tracer Changes from the spec?

Yes, these changes are missing some portions of the proposed spec changes.  The only parts I implemented were those that would specifically help accomplish the targeted use-case:  
consolidate the spans occurring from the time the user has started to navigate to Home being loaded and populated.

> Also, is there supposed to be an activeSpan on this class? I don't see that in the spec

Yes, this is a convenience method that was borrowed from https://developer.lightbend.com/docs/telemetry/current/extensions/opentracing/api.html#globaltracer-active-span and https://github.com/opentracing-contrib/java-agent#creating-custom-rules .
We could probably just include it in our implementation if we don't think we want it in this abstract class.